selenium-webdriver/chrome.d.ts

```diff
@@ -0,0 +1,418 @@
+import * as webdriver from "./index";
+import * as remote from "./remote";
+
+declare namespace chrome {
```

Comment:
This used to appear in a global context. Do we need `export as namespace chrome`?